lisa/tools/lsblk.py

```diff
@@ -120,6 +121,21 @@ class Lsblk(Tool):
     def command(self) -> str:
         return "lsblk"
 
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    def _install(self) -> bool:
+        if isinstance(self.node.os, FreeBSD):
+            self.node.os.install_packages("lsblk")
+        elif isinstance(self.node.os, Posix):
+            self.node.os.install_packages("util-linux")
```

Comment:
please move `isinstance(self.node.os, Posix)` into `if`, it will meet most situation.